packages/flutter/test/widgets/selectable_region_context_menu_tester_web.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'selectable_region_test.dart';
+
+void startTest() {
+  if (!kIsWeb) {
+    return;
```

Comment:
Should this throw instead? Due to the conditional import we cannot enter this code if it's not web. In fact, this code won't even compile outside the web due to the `import 'dart:html'`.